### PR TITLE
Pull request for Business site

### PR DESCRIPTION
**Pull request by dannuzak**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,178 @@
...
+    </header>
+
+    <section id="intro">
+      <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+      Vitae error ratione a dolorem odio sit omnis, iure est optio sunt nulla.
+      Vitae error ratione a dolorem.</p>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <section class= flex-parent>
```
Comment (HIPPIEKICK) [inline code/index.html:31]: Don't forget quotation marks, and there should be no space after the equal sign!

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,178 @@
...
+            method="POST">
+            
+            <div class="form-group">
+            <label for="firstName"></label>
+            <input 
+            id="firstName" 
+            type="text" 
+            placeholder="First name *"
+            name="firstNameValue"
+            required>
```
Comment (HIPPIEKICK) [inline code/index.html:49]: Super that you're giving the attributes new lines. Just remember to indent as well.

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,178 @@
...
+            type="email"
+            placeholder="E-mail*"
+            name="emailValue"
+            required>
+            </div>
+
+            <div class="form-group">
+            <label class="checkbox-label">
+            <input 
+            type="checkbox">
```
Comment (HIPPIEKICK) [inline code/index.html:84]: Don't forget the name attribute :)

File: code/index.html [hunk 4]
```diff
@@ -1,30 +1,178 @@
...
+            </button>
+            </div>   
+          </div>
+        </form>
+    
+      </div> 
+
+
+    <!-- ACCORDION -->
+    <div class="flex-child accordion"><h2>FAQ</h2>
```
Comment (HIPPIEKICK) [inline code/index.html:103]: Nested elements should be given newlines (and be indented)

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,423 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+
+.hero-video video {
+  min-width: 100%;
```
Comment (HIPPIEKICK) [inline code/style.css:59]: This is making your content too wide for mobile:
Skärmavbild 2021-02-16 kl  14 43 49

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Daniela! Nice work with the custom checkbox and the accordion. Have a look at my comments regarding the responsiveness, the indentation of the HTML file and the attributes. Let me know if something is unclear!

Check out the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) as well as the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap.

File: code/index.html (hunk 1 above)
Comment (dannuzak) [reply code/index.html:31]: ok

File: code/index.html (hunk 2 above)
Comment (dannuzak) [reply code/index.html:49]: Done

File: code/index.html (hunk 3 above)
Comment (dannuzak) [reply code/index.html:84]: Done

File: code/index.html (hunk 4 above)
Comment (dannuzak) [reply code/index.html:103]: Done

File: code/style.css (hunk 5 above)
Comment (dannuzak) [reply code/style.css]: Fixed! :)

**Comment by dannuzak** [comment]
Hi Matilda, I made the changes requested. You wanna take a look? :)

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**